### PR TITLE
ENG-460 Fixed ability to remove Resource categories

### PR DESCRIPTION
**Pull request by ffleandro**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/web/resource/ResourcesController.java [hunk 1]
```diff
@@ -76,20 +79,22 @@
     @ApiResponses(value = {
             @ApiResponse(code = 200, message = "OK"),
             @ApiResponse(code = 401, message = "Unauthorized")})
-    @PostMapping("/plugins/cms/assets")
+    @PostMapping(value = "/plugins/cms/assets"/*, consumes = MediaType.MULTIPART_FORM_DATA_VALUE*/)
```
Comment (joewhite101) [inline plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/web/resource/ResourcesController.java:82]: Is that consumes commented out? Remove if so and add a proper comment if one is needed

Comment (ffleandro) [reply plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/web/resource/ResourcesController.java]: Sorry, was committed accidentally.
Cleaning up...

**Comment by entando-service** [comment]
All is Well

**Pull request merged**